component-maps.yml

```diff
@@ -251,6 +251,30 @@ git:
     release_component: true
     independent_component: true
 
+  mender-setup:
+    docker_image:
+    - mender-client-qemu
+    - mender-client-qemu-rofs
```

Comment:
I know that this isn't something we are going to bother fixing now, but I suppose this shouldn't actually be in the rofs images.

Reply:
This will depend on how it is integrated in `meta-mender`. I suspect that we will add it on demo and not exclude it depending on being read-only or not.

Additionally we could have a configuration where `/etc/mender/mender.conf` is a symlink to somewhere (we had a task for this) and then in theory you could use `mender-setup`.

Being an internal testing image, I will leave it as-is :)